### PR TITLE
Add 1.13.2 Deprecations to Deprecations Guide

### PR DESCRIPTION
**Pull request by toddjordan**
Part of Issue https://github.com/emberjs/website/issues/2256

### CODE REVIEW

File: source/deprecations/v1.x.html.md [hunk 1]
```diff
@@ -1023,6 +1023,42 @@ proxying behavior is being used.
...
+```
+and
+```hbs
+{{#each people key='@item' as |person|}}
+...
+{{/each}}
+```
+with
+```hbs
+{{#each people key='@identity' as |person|}}
```
Comment (rwjblue) [inline source/deprecations/v1.x.html.md]: I would prefer to just include the latter example (the `simply` one).

**Comment by rwjblue** [comment]
left one comment, but looks great otherwise!

File: source/deprecations/v1.x.html.md (hunk 1 above)
Comment (toddjordan) [inline source/deprecations/v1.x.html.md]: agreed, will update

**Comment by toddjordan** [comment]
fixt 👍

**Pull request merged**